src/client/routes/Portfolio/index.tsx

```diff
@@ -24,125 +35,147 @@ const HeaderCard = styled(SummaryCard)`
-  // const labsSummary = useAppSelector(LabsSelectors.selectSummaryData);
-  const walletSummary = useAppSelector(TokensSelectors.selectSummaryData);
+  // TODO: pull all selectors in with single import statement
+  const walletIsConnected = useAppSelector(WalletSelectors.selectWalletIsConnected);
+  const userWallet = useAppSelector(WalletSelectors.selectSelectedAddress);
   const userTokens = useAppSelector(TokensSelectors.selectUserTokens);
   const activeModal = useAppSelector(ModalSelectors.selectActiveModal);
   const appStatus = useAppSelector(AppSelectors.selectAppStatus);
   const tokensListStatus = useAppSelector(TokensSelectors.selectWalletTokensStatus);
   const generalLoading = (appStatus.loading || tokensListStatus.loading || isMounting) && !activeModal;
+  const userPortfolio = useAppSelector(LinesSelectors.selectUserPortfolio);
+  const portfolioAddress = userAddress ? userAddress : userWallet;
   const userTokensLoading = generalLoading && !userTokens.length;
-
-  const netWorth = toBN(vaultsSummary.totalDeposits)
-    .plus(walletSummary.totalBalance)
-    // .plus(labsSummary.totalDeposits)
-    .toString();
-
-  const summaryCardItems = [
-    { header: t('dashboard.total-net-worth'), Component: <Amount value={netWorth} input="usdc" /> },
-  ];
-  if (walletIsConnected) {
-    summaryCardItems.push({
-      header: t('dashboard.available-deposit'),
-      Component: <Amount value={walletSummary.totalBalance} input="usdc" />,
-    });
-  }
-  if (currentNetworkSettings.earningsEnabled) {
-    summaryCardItems.push(
-      {
-        header: t('dashboard.vaults-earnings'),
-        Component: <Amount value={vaultsSummary.totalEarnings} input="usdc" />,
-      },
-      {
-        header: t('dashboard.vaults-est-yearly-yield'),
-        Component: <Amount value={vaultsSummary.estYearlyYeild} input="usdc" />,
-      }
-    );
-  }
+  const [portfolioLoaded, setPortfolioLoaded] = useState<boolean>(false);
+  const [currentRole, setRole] = useState<string>(BORROWER_POSITION_ROLE);
+
+  // TODO: Add types
+  const [lenderPositions, setLenderPositions] = useState<CreditPosition[]>([]);
+  const [borrowerPositions, setBorrowerPositions] = useState<CreditPosition[]>([]);
+
+  const setSelectedLine = (address: string) => dispatch(LinesActions.setSelectedLineAddress({ lineAddress: address }));
+  const selectedLine = useAppSelector(LinesSelectors.selectSelectedLine);
+
+  // console.log('user portfolio', userPortfolio, selectedLine);
+  // console.log('user portfolio lines map: ', linesMap);
+
+  const availableRoles = [BORROWER_POSITION_ROLE, LENDER_POSITION_ROLE];
+
+  const SummaryCardItems = availableRoles.map((role: string) => {
+    return {
+      header: t(''),
+      Component: (
+        <RoleOption onClick={() => handleSetRole(role)} active={role === currentRole} key={`s-${role}`}>
+          {t(`settings:${role}`)}
+        </RoleOption>
+      ),
+    };
+  });
+
+  const handleSetRole = (role: string) => {
```

Comment:
Why would we want to reset positions to [] for borrower and lender, wouldn't that mean we need to refetch/setState?